resources/js/components/MobileMenu.vue

```diff
@@ -0,0 +1,106 @@
+<script setup>
+import { Link } from '@inertiajs/inertia-vue3'
+import { HomeIcon, LightBulbIcon, MapIcon, NewspaperIcon } from "@heroicons/vue/24/solid";
+defineProps({
+    user: Object
+})
+
+const items = [
+    {
+        name: 'Home',
+        component: 'Home',
+        url: '/',
+        icon: HomeIcon
+    },
+    {
+        name: 'Lộ trình',
+        component: 'LearningPaths',
+        url: '/learning-paths',
+        icon: MapIcon,
+    },
+    {
+        name: 'Học',
+        url: '/courses',
+        component: 'Courses',
+        icon: LightBulbIcon,
+    },
+    {
+        name: 'Blog',
+        component: 'Blog',
+        url: '/blog',
+        icon: NewspaperIcon
+    },
+]
+</script>
+
+<template>
+        <div class="fixed top-0 left-0 h-[150vh] z-21 w-full bg-[rgba(0,0,0,.3)]">
+            <div class="w-[60%] bg-white z-10 h-full absolute top-0 left-0">
+                <div class="flex flex-col text-[1.6rem] max-h-full overflow-y-auto pt-[16px] pl-[32px]">
```

Comment:
ko chơi set cứng text-[1.6rem] như kia xoá đi, để mặc định thôi
pt 16px là pt-4 (4 * 4=16), pl 32px là 8 (8 * 4=32)

```suggestion
                <div class="flex flex-col max-h-full overflow-y-auto pt-4 pl-8">
```